docs/content/any/guides/policies/index.md

```diff
@@ -9,96 +9,38 @@ aliases:
-
-The purpose of _this_ guide is to serve as a tutorial introduction
-to the Polar language. When you've finished it, you should be able
-to read and write simple Polar rules over your own application objects
-and types.
-
-If you have some prior exposure to logic programming, the Polar
-language will feel very familiar. If not, don't panic! Polar is
-designed to be simple, often allowing more concise
-expression of authorization rules than equivalent code in an
-imperative language. If you can express an authorization rule
-as a simple declarative sentence of English (or other natural language),
-it should be straightforward to turn into a Polar rule.
-
-If you're unsure of what _kind_ of authorization policy to write,
-you may want to head over to the [Conceptual Guides](guides) or
-start with a familiar pattern such as [roles](/guides/rbac).
-This tutorial will not be concerned with a specific kind of policy,
-but rather with the language in which many different kinds of
-policies may be expressed.
-
-## Application Setup
-
-To use Oso for authorization, your application must first:
-
-1. Load the Oso library: `from oso import Oso`
-2. Create an {{% apiDeepLink class="Oso" /%}} instance: `oso = Oso()`
-3. Load a policy: `oso.load_file(policy)`.
-
-Then, at authorization time, you can call:
+# Write Custom Policies
 
-```python
-oso.is_allowed(actor, action, resource)
-```
+In Oso, policies are where authorization logic is stored. Policy files are
+separate from your application code, and are written in a declarative language
+called Polar (file extension `.polar`).
+
+Oso provides policy building blocks for common authorization models like [role-based
+access control](/guides/rbac), [attribute-based access control](/guides/attributes), and
+[resource hierarchies](/guides/hierarchies).
 
-This returns a boolean indicating whether the given `actor`
-is authorized to perform `action` on `resource` according to
-the current policy.
+This guide covers how to extend those building blocks by writing custom
+policies in Oso's policy language, Polar.
+You will learn the basic structure and operations of Polar rules, and how to
+write rules that refer to application classes, instances, and fields.
 
-The arguments to `oso.is_allowed` may be arbitrary Python
-objects. We'll often use strings for examples, but in your
-application they could be ORM model objects, URIs, numbers, etc.
+## What is a policy?
+
+In Oso, policies are where authorization logic is stored. Policy files are
+separate from your application code, and are written in a declarative language
+called Polar (file extension `.polar`).
+
+<!-- TODO: update enforcmeent link once docs are ported -->
+Your application can query a policy using the [Enforcement APIs]({{< ref href="/guides/enforcement" lang="python" >}}) to get authorization decisions.
+The most basic component of a policy is a **rule**, which describes a logical condition that must be true in order for a query to succeed.
```

Comment:
defining a rule as "a logical condition that must be true in order for a query to succeed" feels very abstract and not very related to authorization

Reply:
Taken from old language.
```suggestion
The most basic component of a policy is a **rule**, which describes who is allowed to access what resources in an application.
```